src/katcp_fpga.py

```diff
@@ -65,7 +65,15 @@ def connect(self, timeout=1):
         :return:
         """
         if not self.is_connected():
-            self.start(daemon=True)
+            # Implement backward / forwards compabitlity for change in daemonization APIs
```

Comment:
Weird, I thought we merged this fix a while ago :)  Small spelling mistake: compabitlity -> compatibility